addons/udes_common/models/models.py

```diff
@@ -134,3 +135,43 @@ def selection_display_name(self, selection_field_name):
     # Odoo has a nice function that handles all the weird ways you can define selections!
     selection_values_dict = dict(selection_field._description_selection(self.env))
     return selection_values_dict.get(self[selection_field_name], False)
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
```

Comment:
Needs black formatting, happy for it to be done at the end though.

Reply:
Made the changes